srml/executive/src/lib.rs

```diff
@@ -14,7 +14,55 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Executive: Handles all of the top-level stuff; essentially just executing blocks/extrinsics.
+//! # Executive Module
+//!
+//! The executive module acts as the orchestration layer for the runtime. It dispatches incoming
+//! extrinsic calls to the respective modules in the runtime.
+//!
+//! ## Overview
+//!
+//! The executive module is coupled with the [system module](../srml_system/index.html) and lets the runtime
+//! communicate with the SRML. It is unique from other modules in that it does not incorporate the `decl_event!`,
```

Comment:
I rephrased the overview a bit:

"The executive module is not a typical SRML module providing functionality around a specific feature. It is more like a cross-cutting framework component for the SRML. It includes functions for checking and applying extrinsics, block execution, among other things. It works in conjunction with the SRML system module to perform these cross-cutting functions."